lib/backend.js

```diff
@@ -281,13 +281,10 @@ Backend.prototype._sanitizeSnapshots = function(agent, projection, collection, s
     }
   }
 
-  var request = {
-    collection: collection,
-    snapshots: snapshots,
-    snapshotType: snapshotType
-  };
+  requestContext.collection = collection;
```

Comment:
It might be a good optimisation to initialise these properties whenever `requestContext` is initialised. It may be beneficial to declare a `class` for clarity about what exact information we're expecting in the middleware.